### DISCUSSION TITLE
update() missing 1 required positional argument: 'x' in Statistic Regression

### DISCUSSION THREAD

**Discussion by XaviMartiCamarasa** [online-ml/river#1082] [Q&A] [answered]
Hi I'm trying to use a simple StatisticRegressor.
![image](https://user-images.githubusercontent.com/117338671/200605728-0e6c6235-a5ae-43a7-9942-d99b642447c2.png)
Why I'm getting this error?
![image](https://user-images.githubusercontent.com/117338671/200606096-afb330c4-22e0-4a4e-9088-0bbce4272248.png)
What should I try to "play" with that Serie?? 

Many Thanks (again)

**Comment by VaysseRobin** [comment, marked as answer]
Hi! I think the error comes from this line: `model = dummy.StatisticRegressor(stats.AbsMax)`. You should use `stats.AbsMax()` (with brackets). 
I haven't tried this fix, but I'm pretty sure the error comes from there.

For future discussions, you should provide code that can be copy-pasted instead of images, as it can be very time consuming for us to rewrite all the code based on images in order to debug (you can refer to the "Bug report" problem template for more details).

**Reply by XaviMartiCamarasa** [reply]
You are completely right! Many thanks!
